### ISSUE TITLE
Collect an additional fee for LoopEat

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#2284]
When the order contains LoopEat reusable packagings, an additional fee of `0.90€` should be collected (for washing). 
Ideally, this fee should go directly to LoopEat via Stripe. 

### 1/ Modify order processing

Currently, when adding LoopEats to the order, an adjustment with an amount of zero is added. This is implemented in a "generic" way, by leveraging our internal reusable packaging system. Basically, a `ReusablePackaging` for LoopEat is created, with a price of zero.

https://github.com/coopcycle/coopcycle-web/blob/023f25fb16d01576e78000ae4e49090be772e3e6/src/Controller/Utils/RestaurantTrait.php#L164-L173

The [OrderDepositRefundProcessor](https://github.com/coopcycle/coopcycle-web/blob/023f25fb16d01576e78000ae4e49090be772e3e6/src/Sylius/OrderProcessing/OrderDepositRefundProcessor.php) class is responsible for adding an adjustment for packagings to the order. For LoopEat, it adds an adjustment with an amount of zero. 

This can't be used for the LoopEat processing fee, because the amount of adjustments with type  `REUSABLE_PACKAGING_ADJUSTMENT` goes to the restaurant. 

We need to introduce a new type of adjustment, `LOOPEAT_PROCESSING_FEE`, specifically for this. 
The amount of this adjustment should increase the platform fee. 

So basically, the implementation would be: 
- Add a new order processor, `LoopeatOrderProcessor`, that adds the new adjustment. It must be executed before `OrderFeeProcessor`
- Modify `OrderDepositRefundProcessor`, so that it does nothing when LoopEat is used. 
- Modify `OrderFeeProcessor`, so that it increases the platform fee adjustment, based on the amount of the LoopEat processing fee. 

### 2/ Send fee to LoopEat (optional)

Once the money has been captured, it should be possible to send the money directly to LoopEat's Stripe account using [Transfers](https://stripe.com/docs/connect/charges-transfers)

To achieve this, LoopEat's Stripe account should be connected to the platform's Stripe account using Stripe Connect.
